.changes/next-release/Feature-d124bdaa-a3b7-4d89-84c5-f7cbe495f128.json

```
@@ -0,0 +1,4 @@
+{
+	"type": "Feature",
+	"description": "Upgrade amazon-states-language-service to 1.11. This new version adds Fail State fields ErrorPath and CausePath, and on Retriers MaxDelaySeconds and JitterStrategy."
```

Some extra context will be helpful for customers:

```suggestion
	"description": "Step Functions: Upgrade amazon-states-language-service to 1.11. This new version adds Fail State fields ErrorPath and CausePath, and on Retriers MaxDelaySeconds and JitterStrategy."
```

---

done!